integration_tests/cork_test.go

```diff
@@ -272,7 +293,15 @@ func (s *IntegrationTestSuite) TestCork() {
 			s.T().Logf("got %v logs", len(logs))
 			if len(logs) == 1 {
 				s.T().Log("saw mock function event!")
-				return true
+
+				log := logs[0]
+				if len(log.Data) > 0 {
+					var event AaveV2MockRebalance
+					err := aave_abi.UnpackIntoInterface(&event, "mockRebalance", log.Data)
+					s.Require().NoError(err, "failed to unpack mockRebalance event from log data")
+					s.Require().Equal(event.MinAssetsOut, big.NewInt(1000))
```

Comment:
If you're going to inspect `MinAssetsOut`, is it worth also checking the route and swap values?

Reply:
Seemed unnecessarily complicated to me?